pygmt/tests/test_coast.py

```diff
@@ -14,60 +13,37 @@ def test_coast():
     """
     fig = Figure()
     fig.coast(
-        R="-30/30/-40/40",
-        J="m0.1i",
-        B=5,
-        I="1/1p,blue",
-        N="1/0.25p,-",
-        W="0.25p,white",
-        G="green",
-        S="blue",
-        D="c",
-        A=10000,
+        region="-30/30/-40/40",
+        projection="m0.1i",
```

Comment:
Good opportunity to change to SI units, and use a Python list for `region`.
```suggestion
        region=[-30, 30, -40, 40],
        projection="m0.25c",
```